app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/80958128/a983453e-2622-4199-a001-98944562ec77)
u should use one of this variations indentations for cases like this
dont mix they))